hyperfunctions.cpp

```diff
@@ -755,14 +836,17 @@ void  HyperFunctions::SemanticSegmenter()
             double low_val;
             for (int i=0; i<temp_results.size(); i++)
             {
+
                 if (i==0)
                 {
-                low_val=temp_results[i].at<uchar>(j,k);
-                temp_class_img.at<Vec3b>(Point(k,j)) = reference_colors[i];
+                    low_val=temp_results[i].at<uchar>(j,k);
```

Comment:
why is index not the same low_val and temp_class_img??

Reply:
Wait anthony I don't think I changed anything in this function

Reply:
yeah this was me. It was a reminder to myself to look into